kube/services/portal/portal-deploy.yaml

```diff
@@ -189,6 +189,12 @@ spec:
             GEN3_DATA_UPLOAD_BUCKET|-value: ""-|
           - name: BASENAME
             value: /portal
+          - name: ARGO_TEMPLATE
+            valueFrom:
+              configMapKeyRef:
+                name: manifest-portal
+                key: ARGO_TEMPLATE
```

Comment:
is this variable going to be shared by other services?

if not, why don't just put it into portal config?

Reply:
should not be shared by other services, can you point me to an example env var set in portal config @mfshao

Reply:
cool got it, thanks for the info.